### PR TITLE
Add Koios + Ogmios Java Clients to Builder Tools, plus ogmios + kotlin tags

### PR DESCRIPTION
**Pull request by edridudi**
## Checklist

- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have read the [Builder Tool Requirements](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/builder-tools.js)
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Builder Tool addition

* Title: Koios Java Client
* Description: A Java API library for interacting with Koios Server instances.
* Website:  https://github.com/cardano-community/koios-java-client
* Getstarted: null
* Tags:
  * `koios`
  * `java`
  * `sdk`
  
## Builder Tool addition

* Title: Ogmios Java Client
* Description: A Java API library for interacting with Ogmios.
* Website:  https://github.com/adabox-aio/ogmios-java-client
* Getstarted: null
* Tags:
  * `ogmios`
  * `websocket`
  * `java`
  * `sdk`

### CODE REVIEW

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -270,14 +294,6 @@ export const Tags = {
...
     color: "#4a69BD"
   },
-
-  // Koios
-  koios: {
-    label: "Koios",
-    description: "Koios and its integrations",
-    icon: null,
-    color: "#4d21b8"
-  },
```
Comment (rphair) [inline src/data/builder-tools.js:280]: And what would you think of moving the `koios` tag definition back where it was in the source file, so it doesn't mistakenly look like this part of the code was changed?

**Review by rphair — COMMENTED** [review]
This addition has some pretty significant side effects, including the creation of 2 new categories (`ogmios` and `kotlin`) which should have been mentioned in the title and/or description of this PR.

These would be pretty thin categories so far: not really for `ogmios` because it's Cardano specific but definitely for `kotlin` which looks to me (as a not-really developer) as a flavour of `java` and therefore there's no need for this tag + very little need for the `ogmios` tag.

It would be easy for me to approve adding the tools themselves, but the additional categorisation seems too-specific unless there's the approval of other reviewers with a little more experience in these areas... it's hard from my point of view to anticipate how much the 2 new categories would satisfy the need for expansion in the long term.

So @edridudi I'll tag a few more reviewers for now & we'll see what they feel about the categorisation issue.

File: src/data/builder-tools.js (hunk 1 above)
Comment (edridudi) [reply src/data/builder-tools.js:280]: Yeah, I was wondering whether the new categories will cause any disagreements but that's how I feel is the right way to represent both Ogmios and Koios Java tools as best as we can.
Ogmios category will get bigger as we have already 2 more language-specific clients to interact with ogmios. so basically it's a big category of 5 tools including Ogmios Server. (https://ogmios.dev/typescript-client/)
I can add the other 2 as well but I'm not their maintainer.

Kotlin is a programming language. you can say it looks similar or resembles Java but in fact, it's a different programming language. Personally, I am familiar with Java but not with Kotlin and not with the syntax.

I noticed that this file was created in alphabetic order so I just wanted to give respect to the creator's first intention.
The alphabetic order wasn't kept as PR's kept merging. 
It won't be a problem to change it back if you insist.

**Comment by edridudi** [comment]
Yeah, I was wondering whether the new categories will cause any disagreements but that's how I feel is the right way to represent both Ogmios and Koios Java tools as best as we can.
Ogmios category will get bigger as we have already 2 more language-specific clients to interact with ogmios. so basically it's a big category of 5 tools including Ogmios Server. (https://ogmios.dev/typescript-client/)
I can add the other 2 as well but I'm not their maintainer.

Kotlin is a programming language. you can say it looks similar or resembles Java but in fact, it's a different programming language. Personally, I am familiar with Java but not with Kotlin and not with the syntax.

File: src/data/builder-tools.js (hunk 1 above)
Comment (rphair) [reply src/data/builder-tools.js:280]: that's OK @edridudi I won't insist :)

**Review by rdlrt — APPROVED** [review]

**Comment by rdlrt** [comment]
I agree that ogmios has growing projects coming in (mentioned 5 for starters).

As for Kotlin, I am a bit neutral - I agree with @rphair that it's currently quite thin. Having said that I suppose it will eventually have it's place regardless being one of the most common languages used for mobile frameworks. Hence - from site user point of view, I do find it useful for incoming user who might not be familiar with ecosystem to see the filter, even if there aren't a lot of projects atm

**Review by rphair — APPROVED** [review]

**Review by fill-the-fill — APPROVED** [review]

**Pull request merged**